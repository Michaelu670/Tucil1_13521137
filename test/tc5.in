1 2 2 1
A 2 3 4 5 6 7 8
A A
A A A B
21 A B C
A A A A