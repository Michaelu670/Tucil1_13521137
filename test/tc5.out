Kartu : A A A A
no solution found
Execution time = 0 milliseconds
